src/queries/author/index.js

```diff
@@ -6,9 +6,9 @@ import { AuthorType } from '../../types/author';
 
 const AuthorData = {
   id: 1,
-  firstName: 'haloo',
-  lastName: 'haloo',
-  posts: 'haloo',
+  firstName: 'Ihsan',
```

Comment:
I thing it's unnecessary changes..please remove it

Reply:
Why do you think 'Ihsan' is not a common first name? :'(

Reply:
Because it's not a human name.